client/src/app/pages/profile/profile.component.html

```diff
@@ -1,3 +1,16 @@
 <app-header></app-header>
 <app-transaction-history class="transaction-history-container" [currentProfileUser]="currentProfileUser$ | async">
-</app-transaction-history>
\ No newline at end of file
+
+</app-transaction-history>
+<div class="followers-follwings-container" *ngIf="completeUserProfile">
+
+    <button mat-button class="follow-unfollow-label" (click)="openDialog('followers')"
+        [disabled]="getFollowers().length===0">{{getFollowers().length}}
+        followers</button>
+    <button mat-button class="follow-unfollow-label" (click)="openDialog('followings')"
+        [disabled]="getFollowings().length===0">{{getFollowings().length}}
```

Comment:
we shouldn't disable the button if they don't have any followers, we can open up the popup and say "hey you aren't following anyone"